src/SpotifySongSearch.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from "react";
+import { getToken } from "./utils/tokenStorage";
+import { fetchSpotifyTracksIds } from "./utils/spotify";
+
+const parseArtistsTracks = (text) => {
+  return text.split("\n").map((artistTrackStr) => {
+    const splitArtistTrack = artistTrackStr.split(" - ");
+
+    if (splitArtistTrack.length < 2)
+      return { artist: splitArtistTrack[0].trim(), track: "" };
+
+    return {
+      artist: splitArtistTrack[0].trim(),
+      track: splitArtistTrack[1].trim(),
+    };
+  });
+};
+
+const TrackView = ({ track, key }) => {
+  return <li key={key}>{track.trackId}</li>;
+};
+
+const ErrorView = ({ error, key }) => {
+  return <li key={key}>There was an error: {error.message}</li>;
+};
+
+const ResponsesView = ({ responses }) => {
+  const views = responses.map((response, idx) => {
+    if (!response.hasError) return <TrackView track={response} key={idx} />;
```

Comment:
You can use ternary operator here instead of 'if'

Reply:
But what is the difference? Prettier reformats this code with ternary operator like if it has "if" statement.